src/get-member/components/index/api.ts

```diff
@@ -0,0 +1,37 @@
+import { GetMemberResponse} from "./interfaces.ts";
+
+const GENERIC_ERROR_ID = "error"
+
+const MOCK_MEMBER_1: GetMemberResponse = {
+    member_id: "2023-2024 001",
+    name: "member",
+    email: "member@ubcani.com",
+    student_number: "123456789",
+    join_year: "2021-2022"
+}
+
+const MOCK_MEMBER_2: GetMemberResponse = {
+    member_id: "2023-2024 002",
+    name: "member2",
+    email: "member2@ubcani.com",
+    student_number: "987654321",
+    join_year: "2022-2023"
+}
+
+const MOCK_MAP: ReadonlyMap<string, GetMemberResponse> = new Map([
+    [MOCK_MEMBER_1.member_id, MOCK_MEMBER_1],
+    [MOCK_MEMBER_2.member_id, MOCK_MEMBER_2]
+]);
+
+export const getMemberApiCall = (memberId: string): Promise<GetMemberResponse> => {
```

Comment:
Returns promise like actual API call would, so the rest of the code should work fine when we swap from mocked to actual calls, as long as we keep this interface the same